config/database.js

```diff
@@ -15,7 +15,7 @@ module.exports = ({ env }) => ({
       connector: 'mongoose',
       settings: {
         database: env('DATABASE_NAME'),
-        uri: `mongodb+srv://${env('DATABASE_USERNAME')}:${env('DATABASE_PASSWORD')}@${env('DATABASE_HOST')}/${env('DATABASE_NAME')}?retryWrites=true&w=majority`,
+        uri: `mongodb://${env('DATABASE_USERNAME')}:${env('DATABASE_PASSWORD')}@${env('DATABASE_HOST')}:${env('DATABASE_PORT')}/?ssl=true&replicaSet=globaldb&retrywrites=false&maxIdleTimeMS=120000`,
```

Comment:
@Alihaider168 

- why do we need replicaSet ?
- Why retry write to false?
- Why 2 minutes timeout?
The last 2 Flags are network-dependent. If you set those values only because you were having performance issue in the development then set those only for the dev environment.